{
    "comments": [
        {
            "author": "Thomas Mueller",
            "body": "I don't know it which version it broke. \nThere seem to be no test cases.",
            "date": "2011-02-07T12:56:53.890+0000",
            "id": 0
        },
        {
            "author": "Jukka Zitting",
            "body": "Hmm, this looks like a regression from JCR-2832.",
            "date": "2011-02-07T13:48:31.716+0000",
            "id": 1
        },
        {
            "author": "Thomas Mueller",
            "body": "It looks like the exception is swallowed.",
            "date": "2011-02-07T14:24:39.077+0000",
            "id": 2
        },
        {
            "author": "Thomas Mueller",
            "body": "Committed in the 2.2 branch and trunk.",
            "date": "2011-02-07T15:48:30.710+0000",
            "id": 3
        }
    ],
    "component": "jackrabbit-core",
    "description": "Using the database journal (any database) fails with the following stack trace:\n\njava.lang.IllegalStateException: already in batch mode\n\tat org.apache.jackrabbit.core.util.db.ConnectionHelper.startBatch(ConnectionHelper.java:212)\n\tat org.apache.jackrabbit.core.journal.DatabaseJournal.doSync(DatabaseJournal.java:449)\n\tat org.apache.jackrabbit.core.journal.AbstractJournal.lockAndSync(AbstractJournal.java:254)\n\tat org.apache.jackrabbit.core.journal.DefaultRecordProducer.append(DefaultRecordProducer.java:51)\n\tat org.apache.jackrabbit.core.cluster.ClusterNode$WorkspaceUpdateChannel.updateCreated(ClusterNode.java:539)\n\tat org.apache.jackrabbit.core.state.SharedItemStateManager$Update.begin(SharedItemStateManager.java:559)\n\tat org.apache.jackrabbit.core.state.SharedItemStateManager.beginUpdate(SharedItemStateManager.java:1457)\n\tat org.apache.jackrabbit.core.state.SharedItemStateManager.update(SharedItemStateManager.java:1487)\n\tat org.apache.jackrabbit.core.state.LocalItemStateManager.update(LocalItemStateManager.java:351)\n\tat org.apache.jackrabbit.core.state.XAItemStateManager.update(XAItemStateManager.java:354)",
    "hasPatch": false,
    "hasScreenshot": false,
    "id": "JCR-2882",
    "issuetypeClassified": "BUG",
    "issuetypeTracker": "BUG",
    "priority": "Major",
    "product": "JACKRABBIT",
    "project": "JACKRABBIT",
    "summary": "DatabaseJournal: java.lang.IllegalStateException: already in batch mode",
    "systemSpecification": true,
    "version": ""
}